CHANGELOG.D/2297.feature

```diff
@@ -0,0 +1,3 @@
+Added `neuro blob importbucket` command to import external bucket. External buckets support
```

Comment:
Is the changelog actual?

Reply:
Sorry, I did not rebase the PR. Fixed now.